### ISSUE TITLE
result on AV2

### ISSUE THREAD

**Issue by weihuaLi1** [Tsinghua-MARS-Lab/DenseTNT#30]
Argoverse 2 dataset has been released. Have you ever tried to use DenseTNT on it? And what about the result?

**Comment by GentleSmile** [comment]
We haven't tested it on Argoverse 2 yet. If it goes well, we will update results on Argoverse 2 in August.

**Comment by yanwu-ge** [comment]
Hello, I am also interested in the results on Argoverse 2. Have you finished the work yet ? When will it be released ? Looking forward to your reply.

**Comment by GentleSmile** [comment]
We are tuning DenseTNT on Argoverse 2. We add the training code of Argoverse 2 for reference: https://github.com/Tsinghua-MARS-Lab/DenseTNT/tree/argoverse2

**Comment by sjx5978** [comment]
> We are tuning DenseTNT on Argoverse 2. We add the training code of Argoverse 2 for reference: https://github.com/Tsinghua-MARS-Lab/DenseTNT/tree/argoverse2

Hello, Thank you for your wonderful work.
I try to use your code to train DenseTNT on Argoverse 2, and there was no problem with the training process. But when I started starting the validation phase(i.e. add --do_eval --do_eval --eval_params optimization MRminFDE=0.0 cnt_sample=9 opti_time=0.1), an error will occur. May I ask that the author's code on Argoverse 2 validation set hasn't been released yet? Looking forward to your reply.
<img width="931" alt="截屏2023-03-02 16 01 39" src="https://user-images.githubusercontent.com/59856005/222367433-252098b7-7b3e-4654-abe1-8b3d49b1a070.png">

**Comment by Xinchengzelin** [comment]
> We are tuning DenseTNT on Argoverse 2. We add the training code of Argoverse 2 for reference: https://github.com/Tsinghua-MARS-Lab/DenseTNT/tree/argoverse2

Hi,  do you have plans for  updating the Argoverse2 result?  
Have you finished the branch argoverse2? I mean the code could run

**Comment by GentleSmile** [comment]
Hi, we have just added evaluation metrics for Argoverse2.

**Comment by GentleSmile** [comment]
> > We are tuning DenseTNT on Argoverse 2. We add the training code of Argoverse 2 for reference: https://github.com/Tsinghua-MARS-Lab/DenseTNT/tree/argoverse2
> 
> Hi, do you have plans for updating the Argoverse2 result? Have you finished the branch argoverse2? I mean the code could run

Hi, we have just added evaluation metrics for Argoverse2.

**Comment by Xinchengzelin** [comment]
> > > We are tuning DenseTNT on Argoverse 2. We add the training code of Argoverse 2 for reference: https://github.com/Tsinghua-MARS-Lab/DenseTNT/tree/argoverse2
> > 
> > 
> > Hi, do you have plans for updating the Argoverse2 result? Have you finished the branch argoverse2? I mean the code could run
> 
> Hi, we have just added evaluation metrics for Argoverse2.

Could I ask a simple question? If we use Argeverse2 dataset,  do we also need Argoverse1 API?  Or we just need av2-api?

**Comment by GentleSmile** [comment]
> > > > We are tuning DenseTNT on Argoverse 2. We add the training code of Argoverse 2 for reference: https://github.com/Tsinghua-MARS-Lab/DenseTNT/tree/argoverse2
> > > 
> > > 
> > > Hi, do you have plans for updating the Argoverse2 result? Have you finished the branch argoverse2? I mean the code could run
> > 
> > 
> > Hi, we have just added evaluation metrics for Argoverse2.
> 
> Could I ask a simple question? If we use Argeverse2 dataset, do we also need Argoverse1 API? Or we just need av2-api?

We only need to install av2 using pip
